platform/broadcom/sonic-platform-modules-juniper/README.md

```diff
@@ -1 +1,291 @@
+```
+
+You can find the correct "/dev/sdX" by validating the "dmesg" output after inserting an USB device into the Linux workstation.
+
+
+1) Booting from a USB Memory Device
+
+To boot from an external USB memory device connected to the QFX5210-64C-S switch, you need to:
+
+a. Insert the USB memory device to the USB port of the QFX5210-64C-S switch.
+
+b. Power on the QFX5210-64C-S switch and enter the BIOS configuration by pressing the Esc key, as displayed in the console screen.
+
+c. Set the hard drive boot order as follows:
+
+   When you see the "Boot Option #1" displayed on the console screen, select the USB memory device:
+
+    Boot-->Boot Option Priorities-->Boot Option #1
+
+   If the USB memory device name is not listed in "Boot Option #1", check the priorities in the hard drive boot order:
+
+    Boot-->Hard Drive BBS Priorities-->Boot Option #1
+
+
+   For example, consider "JetFlashTranscend 8GB 8.07" as the USB memory device, the boot order will display as follows:
+
+
+    Boot Option Priorities
+
+    Boot Option #1          [JetFlashTranscend 8...]
+
+    Boot Option #2          [ATP ATP IG eUSB 1100]
+
+    Boot Option #3          [IBA GE Slot 00A0 v1543]
+
+    Boot Option #4          [UEFI: Built-in EFI ...]
+
+
+d. Go to "Save & Exit" in the BIOS screen and from the Boot Override option select the USB memory device (For example, JetFlashTranscend 8GB 8.07).
+
+e. After a few seconds, the QFX5210-64C-S switch would restart and boot from the USB memory device and then you will see the following on the console screen:
+
+
+                     GNU GRUB  version 2.02~beta2+e4a1fe391
+
+
+
+ +----------------------------------------------------------------------------+
+
+ |*ONIE: Rescue                                                               |
+
+ | ONIE: Embed ONIE                                                           |
+
+ |                                                                            |
+
+ |                                                                            |
+
+ |                                                                            |
+
+ |                                                                            |
+
+ |                                                                            |
+
+ |                                                                            |
+
+ |                                                                            |
+
+ |                                                                            |
+
+ |                                                                            |
+
+ |                                                                            |
+
+ +----------------------------------------------------------------------------+
+
+
+
+      Use the ^ and v keys to select which entry is highlighted.
+
+      Press enter to boot the selected OS, `e' to edit the commands before booting or `c' for a command-line.
```

Comment:
Surround this in triple backticks to ensure it is displayed in monospaced font.

Reply:
done. Please take a look

Reply:
Thanks. However, this output appears to have extra blank lines every other line (lines 141, 143, 145, ...). Can you please remove them?